modules/nf-core/kofamscan/tests/main.nf.test.snap

```diff
@@ -0,0 +1,80 @@
+{
+    "test_kofamscan_tsv": {
+        "content": [
+            {
+                "0": [
+                    [
+                        {
+                            "id": "test"
+                        },
+                        "test.kofamscan.txt:md5,63de211bf767ccff9c8bce622a8c26b6"
+                    ]
+                ],
+                "1": [
+
+                ],
+                "2": [
+                    "versions.yml:md5,27e1b383c38beae2cdebb2cc79ff061e"
+                ],
+                "tsv": [
+
```

Comment:
Per the above note about missing ext.args you can see that the tsv output channel is empty even though this purports to be the tsv test.